sonar-scanner-engine/src/main/java/org/sonar/batch/issue/IssueFilters.java

```diff
@@ -52,7 +52,7 @@ public IssueFilters(Project project) {
     this(project, new IssueFilter[0], new org.sonar.api.issue.batch.IssueFilter[0]);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "IssueFilter"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)